app/src/test/java/net/bible/service/format/osistohtml/osishandlers/OsisToCopyTextSaxHandlerTest.kt

```diff
@@ -0,0 +1,92 @@
+package net.bible.service.format.osistohtml.osishandlers
+
+import org.junit.Before
+import org.junit.Test
+
+import org.hamcrest.CoreMatchers.equalTo
+import org.junit.Assert.assertThat
+import org.xml.sax.helpers.AttributesImpl
+
+class OsisToCopyTextSaxHandlerTest {
+
+    private var osisToCopyTextSaxHandlerSingleVerse: OsisToCopyTextSaxHandler? = null
+    private var osisToCopyTextSaxHandlerMultipleVerses: OsisToCopyTextSaxHandler? = null
```

Comment:
I think you could use lateinit variables

Reply:
Done.